src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  height: 15px;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+
+    &:nth-last-child(n + 2) {
+      background-image: url(../images/star-active.svg);
```

Comment:
It would be better to use @for loop in case if we add, for example, a class with the number of active stars like: class="stars stars--count-4". Then we can change the name of the class (like "stars--count-2") and will have only 2 active stars.